com.unity.netcode.gameobjects/Runtime/Messaging/Messages/NetworkTransformMessage.cs

```diff
@@ -81,8 +94,20 @@ public bool Deserialize(FastBufferReader reader, ref NetworkContext context, int
             }
             else
             {
-                // Deserialize the state
-                reader.ReadNetworkSerializableInPlace(ref State);
+                ownerAuthoritativeServerSide = networkManager.DAHost;
```

Comment:
While reviewing your PR, I noticed a bug in DAHost mode.
Just including the fix for that here.